packages/query-core/src/types.ts

```diff
@@ -338,31 +338,17 @@ export interface ResultOptions {
   throwOnError?: boolean
 }
 
-export interface RefetchPageFilters<TPageData = unknown> {
-  refetchPage?: (
-    lastPage: TPageData,
-    index: number,
-    allPages: TPageData[],
-  ) => boolean
-}
-
 export interface RefetchOptions extends ResultOptions {
   cancelRefetch?: boolean
 }
 
-export interface InvalidateQueryFilters<TPageData = unknown>
-  extends QueryFilters,
-    RefetchPageFilters<TPageData> {
+export interface InvalidateQueryFilters extends QueryFilters {
   refetchType?: QueryTypeFilter | 'none'
 }
 
-export interface RefetchQueryFilters<TPageData = unknown>
-  extends QueryFilters,
-    RefetchPageFilters<TPageData> {}
+export interface RefetchQueryFilters extends QueryFilters {}
 
-export interface ResetQueryFilters<TPageData = unknown>
-  extends QueryFilters,
-    RefetchPageFilters<TPageData> {}
+export interface ResetQueryFilters extends QueryFilters {}
```

Comment:
do we need those extra interfaces if they are now exactly the same as `QueryFilters` ? I think I only made them for that feature ...

Reply:
Sure let me remove this one.

Reply:
Done in #b338f7d5f163edc313ab9b44c682ab2aa01f3440
Still some type checks failed in vue tests..

Reply:
All checks have passed.